### DISCUSSION TITLE
Text selection in dark theme seems not highlight

### DISCUSSION THREAD

**Discussion by ParkerOu** [community/community#52618] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Theme: Dark dimmed

https://user-images.githubusercontent.com/25717144/231440194-8a3095d8-d03a-4060-b012-36062dbb8c8b.mp4

**Comment by adenvt** [comment]
It's happen on all dark theme. I switch Light theme for now :')

**Comment by nick-delirium** [comment]
Can't believe it hasn't been fixed for more than a month. Does it take that long to fix simple css line? Are they laid off all UI devs?

**Reply by adenvt** [reply]
> Can't believe it hasn't been fixed for more than a month. Does it take that long to fix simple css line? Are they laid off all UI devs?

It fixed now

**Reply by ParkerOu** [reply]
Disable cache still no highlight effect.

**Reply by nick-delirium** [reply]
Same for me, not fixed. Even in inputs.

**Comment by chrizza87** [comment, marked as answer]
i followed this discussion: https://github.com/orgs/community/discussions/50295

disabling experimental features helped for me: chrome://flags/#enable-experimental-web-platform-features
